frontends/web/src/components/rates/rates.tsx

```diff
@@ -33,7 +33,9 @@ export type Coin = MainnetCoin | TestnetCoin;
 export type Fiat = 'USD' | 'EUR' | 'CHF' | 'GBP' | 'JPY' | 'KRW' | 'CNY' | 'RUB' | 'CAD' | 'AUD';
 
 export type Rates = {
+    /* eslint no-unused-vars: "off" */
     [coin in MainnetCoin]: {
```

Comment:
I tried googling `typescript in` :sweat_smile: what does `in` do in TS?

Reply:
Eslint doesn't understand this and errors, but once we enable the commented out plugin:typescript-eslint/recommended 🤞 we can remove the eslint ignore comment.

Mapped types 
- https://www.typescriptlang.org/docs/handbook/advanced-types.html#mapped-types
- https://mariusschulz.com/blog/mapped-types-in-typescript#more-examples-for-mapped-types

Reply:
We don't need to disable the eslint rule here once `"plugin:@typescript-eslint/recommended"` is enabled

Reply:
Thx for the reference. Seems like uppercase for the type var might be more appropriate, e.g. `Coin in MainnetCoin`.